firestore/query_test.go

```diff
@@ -650,6 +650,72 @@ func createTestScenarios(t *testing.T) []toProtoScenario {
 				},
 			},
 		},
+		{
+			desc: `q.Where("a", ">", 5).FindNearest float64 vector`,
+			in: q.Where("a", ">", 5).
+				FindNearest("embeddedField", []float64{100, 200, 300}, 2, DistanceMeasureEuclidean, nil).q,
```

Comment:
Do we need `ToVector([]float64{100, 200, 300})` here as in L689?

Reply:
The function signature has been changed. PTAL.

PR description has been updated as well